src/ashell/jerry-code.c

```diff
@@ -15,18 +15,18 @@
 #include <zephyr.h>
 
 // JerryScript includes
-#include "../zjs_file_utils.h"
+#include "zjs_file_utils.h"
 #include "jerry-code.h"
 #include "jerryscript-port.h"
 
 // ZJS includes
-#include "../zjs_buffer.h"
-#include "../zjs_callbacks.h"
-#include "../zjs_ipm.h"
-#include "../zjs_modules.h"
-#include "../zjs_sensor.h"
-#include "../zjs_timers.h"
-#include "../zjs_util.h"
+#include "zjs_buffer.h"
+#include "zjs_callbacks.h"
+#include "zjs_ipm.h"
+#include "zjs_modules.h"
+#include "zjs_sensor.h"
+#include "zjs_timers.h"
+#include "zjs_util.h"
```

Comment:
Nice fix, I take it you added it to the path - why didn't we do this before. :)

Reply:
because i told him so :)